### ISSUE TITLE
Remove/unblock the domain nsupdate.info

### ISSUE THREAD

**Issue by Vertux** [badmojr/1Hosts#1794]
<br>
<h2 align="center">Submit Form</h2>
<div align="center">
  Get your issue resolved quickly! Fill in the form accurately.
</div>

**Lists in use:**
- [ ] mini
- [ ] Lite
- [x] Pro
- [ ] Xtra
<br>

**Client:**
AdGuardHome

<br>

**Domains:**
```
*.nsupdate.info
```
<br>

**Details:**
The domain operates a well-respected free [open source DynDNS service](https://github.com/nsupdate-info/nsupdate.info). 
Check out: https://www.nsupdate.info/ for more details.

<div align="center">
  <h2>Thank you for supporting 1Hosts.</h2>
  <sub>It’s people like you who make these lists great! ❤<ub>
</div>

**Comment by spirillen** [comment]
you mean like `pypal-anothersystem.nsupdate.info` and `ipv4.nsupdate.info`? phishing and tracker?

**Comment by spirillen** [comment]
## Search result from External Hosts-Sources
@mypdns's External Hosts-Sources can be found [here](https://github.com/external-sources/hosts-sources?tab=readme-ov-file#external-sources)

```css
data/ShadowWhispererDynamic.txt:nsupdate.info
data/UltimateHostsBlacklist1.txt:pypal-anothersystem.nsupdate.info
data/badmojr1Hosts.txt:ipv4.nsupdate.info
data/badmojr1Hosts.txt:nsupdate.info
```

#### Sorted result

```css
ipv4.nsupdate.info
pypal-anothersystem.nsupdate.info
```

### Search result from easylist

```css
```

## Search in [Matrix](https://mypdns.org/)


Search results from [Matrix](https://mypdns.org) blacklist project

```css
```


Found these RPZ records from My Privacy DNS
```sql
id      domain records  type    content
3018917 nsupdate.info   CNAME   .
```

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
\+ Thanks to [My Privacy DNS](https://mypdns.org/) for this knowledge
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

**Comment by Vertux** [comment]
> you mean like `pypal-anothersystem.nsupdate.info` and `ipv4.nsupdate.info`? phishing and tracker?

I have been using the nsupdate.info service for years, so I am mainly talking about www.nsupdate.info and my own subdomains that I use for DynDNS on nsupdate.info. I guess when you run such a DynDNS service, like Thomas Waldmann does, there is always abuse, but you can inform him (info @ nsupdate.info) and he will suspend those accounts as soon as possible.

nsupdate.info offers three domain alternatives for DynDNS
<img width="953" alt="Bildschirmfoto 2024-06-23 um 14 15 59" src="https://github.com/badmojr/1Hosts/assets/10457918/9c7e876d-3da0-4c43-811a-7da9e4b9a019">

What I'm saying is that the list should only selectively block subdomains that are proven to be malicious.

**Comment by spirillen** [comment]
Ok, it just your request to delete ALL sub-domains by `*.ns...`

which leads to misunderstandings

**Comment by Vertux** [comment]
> Ok, it just your request to delete ALL sub-domains by `*.ns...`
> 
> which leads to misunderstandings

Yes, you are right, that was my fault, I hope I could make it a bit clearer.
